Python/structmember.c

```diff
@@ -243,15 +246,15 @@ PyMember_SetOne(char *addr, PyMemberDef *l, PyObject *v)
             if (ulong_val == (unsigned long)-1 && PyErr_Occurred()) {
                 return -1;
             }
-            *(unsigned int*)addr = (unsigned int)ulong_val;
+            FT_ATOMIC_STORE_UINT_RELEASE(*(unsigned int *)addr, (unsigned int)ulong_val);
             if (ulong_val > UINT_MAX) {
                 WARN("Truncation of value to unsigned int");
             }
         }
         break;
     }
     case Py_T_LONG:{
-        *(long*)addr = PyLong_AsLong(v);
+        FT_ATOMIC_STORE_LONG_RELEASE(*(long*)addr, PyLong_AsLong(v));
         if ((*(long*)addr == -1) && PyErr_Occurred())
```

Comment:
This pattern is broken -- the error check needs to happen before the store to `addr`.

There are a few more instances of this in this file.

Reply:
Yeah, I saw that, but thought not to change the behavior.
I'll fix that.

I guess we would need a news entry then.